src/_zkapauthorizer/storage_common.py

```diff
@@ -43,27 +41,27 @@ class MorePassesRequired(Exception):
         passes indicating passes which failed the signature check.
     """
 
-    valid_count = attr.ib(validator=attr.validators.instance_of((int, long)))
-    required_count = attr.ib(validator=attr.validators.instance_of((int, long)))
+    valid_count = attr.ib(validator=attr.validators.instance_of(int))
+    required_count = attr.ib(validator=attr.validators.instance_of(int))
     signature_check_failed = attr.ib(converter=frozenset)
 
 
 def _message_maker(label):
     def make_message(storage_index):
-        return u"{label} {storage_index}".format(
+        return "{label} {storage_index}".format(
             label=label,
-            storage_index=b64encode(storage_index),
+            storage_index=b64encode(storage_index).decode("utf-8"),
         )
 
     return make_message
```

Comment:
I guess it would be a wire-incompatible change, but I'd be inclined to change this to be `bytes`, rather that `str` here.

Reply:
It turns out that all of the callers take the result and encode it to bytes - so I made this return bytes and simplified all of the callers.